emission/analysis/modelling/tour_model/cluster_pipeline.py

```diff
@@ -75,7 +75,7 @@ def cluster(data, bins):
     min = bins
     max = int(math.ceil(1.5 * bins))
     feat.cluster(min_clusters=min, max_clusters=max)
-    feat.map_clusters()
+    #feat.map_clusters()

```

Comment:
@joshzarrabi, add a comment explaining why it is commented out, or just remove it.
Can be addressed in the next checkin.